deployer/src/persistence/mod.rs

```diff
@@ -210,7 +208,7 @@ impl Persistence {
             .bind(deployment.service_id.to_string())
             .bind(deployment.state)
             .bind(deployment.last_update)
-            .bind(deployment.address.map(|socket| socket.to_string()))
+            .bind(Option::<String>::None)
```

Comment:
Why don't we just remove the address field from the DB?

Reply:
*"for historical purposes..."*
(My thinking is that they'll all get cleaned up in future migrations anyways.)

Reply:
Haha, I was thinking that removing them now makes future migrations (especially the big one) easier :laughing: